src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -66,6 +66,7 @@ public function __construct()
             new ServiceLocatorTagPass(),
             new DecoratorServicePass(),
             new CheckDefinitionValidityPass(),
+            new ResolveAutowireInlineAttributesPass(),
```

Comment:
Any reason why you put that one here? I would register it just before ResolveChildDefinitionsPass, so that resulting definitions can have a parent.

Reply:
No there was no specific reason to it.
I just thought putting it next to `AutowirePass` may have made sense since the order here doesn't matter anyway since it's `krsort` [here](https://github.com/symfony/symfony/blob/dd16390a5290a9a08b2c997fa9d959296bb6f0e0/src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php#L177-L180) anyway :shrug:.

I've moved it now before `ResolveChildDefinitionsPass`.

Reply:
Done [here](https://github.com/symfony/symfony/compare/dd16390a5290a9a08b2c997fa9d959296bb6f0e0..c94322321acafdbaf32044e58b213223645497ce)